pkg/controller/volume/attachdetach/attach_detach_controller.go

```diff
@@ -210,6 +221,11 @@ func (adc *attachDetachController) Run(stopCh <-chan struct{}) {
 	defer runtime.HandleCrash()
 	glog.Infof("Starting Attach Detach Controller")
 
+	if !kcache.WaitForCacheSync(stopCh, adc.podsSynced, adc.nodesSynced, adc.pvcsSynced, adc.pvsSynced) {
```

Comment:
This didn't wait before, did it?  Waiting isn't always required.  I don't know whether this one does or doesn't require the wait.